scheduler/match.go

```diff
@@ -47,10 +52,54 @@ func MemoryAvailable(v float64) ogle.Matcher {
 	return &resourceMatcher{"mem", v}
 }
 
+func (m *attributeMatcher) Matches(o interface{}) (err error) {
+	offer := o.(*mesos.Offer)
+	matched := int(0)
+
+	for _, constraint := range m.SlaveConstraints {
```

Comment:
Rather than having the matcher loop over constraints I think it would be fitting to combine multiple matchers looking for a single. constraints with the AllOf matcher. Just as a code clarity thing the exposed api of AttributeMatch can remain as is.